src/ObjCRuntime/Class.cs

```diff
@@ -65,8 +65,8 @@ public Class (string name)
 		{
 			this.handle = objc_getClass (name);
 
-			if (this.handle == IntPtr.Zero)
-				throw new ArgumentException (String.Format ("'{0}' is an unknown class", name));
+			if (handle == NativeHandle.Zero)
+				ObjCRuntime.ThrowHelper.ThrowArgumentException (nameof (name), "Unknown class");
```

Comment:
This won't show the name of the unknown class anymore (it will say that "name" is an unknown class).